201-web-app-ase-create/azuredeploy.json

```diff
@@ -0,0 +1,172 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aseName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the App Service Environment"
+            }
+        },
+        "aseLocation": {
```

Comment:
If ASE can be deployed in all Azure regions, it might be better to eliminate this parameter and create a variable location with value resourceGroup().location and use it for the resources